app/controllers/answers_controller.rb

```diff
@@ -0,0 +1,55 @@
+class AnswersController < ApplicationController
+  before_action :set_question
+  before_action :set_answer, only: [:show, :update, :destroy]
+
+  def index
+    @answers = @question.answers.all
+
+    render json: @answers
+  end
+
+  def show
+    render json: @answer
+  end
+
+  def create
+    @answer = @question.answers.new(answer_params)
+
+    if @answer.save
+      render json: @answer, status: :created, location: question_answer_path(@question, @answer)
+    else
+      render json: @answer.errors, status: :unprocessable_entity
+    end
+  end
+
+  def update
+    if @answer.update(answer_params)
+      head :no_content
+    else
+      render json: @answer.errors, status: :unprocessable_entity
```

Comment:
@andela-oojewale raised concerns about returning form errors explicity.  Is there a better a way of doing this?